src/types/sequence.rs

```diff
@@ -3,10 +3,11 @@ use crate::exceptions::PyTypeError;
 #[cfg(feature = "experimental-inspect")]
 use crate::inspect::types::TypeInfo;
 use crate::internal_tricks::get_ssize_index;
+use crate::prelude::*;
```

Comment:
I am not keen on using our own prelude inside our fundamental implementations. I think it is too easy to have circular dependencies this way.